mmv1/templates/terraform/custom_flatten/secret_version_access.go.tmpl

```diff
@@ -12,6 +12,9 @@
 */ -}}
 func flatten{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
 	transformed := make(map[string]interface{})
+	if d.Get("secret_data").(string) == "" {
```

Comment:
Are we able to do an existence check on the wo field instead of a non-existence check on the sensitive field? (I am almost certain either another reviewer or myself have asked as much- the explanation has not stuck for me, so asking again to have it reexplained again!)

also nit: Can you comment here to explain why we're not checking the value? Similar to L16/L19 (left/right side line) below.

Reply:
we aren't able to do this since on `ReadResource` the RawConfigAt of a write-only attribute will always return Null. So the check is for the sensitive field itself. We can change this to check for `secret_data_wo_version` instead since that must be set with write-only attribute.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/13135/commits/eb7578b6f56aeadd75fc89a468aa0aed189711a5